.travis.yml

```diff
@@ -16,6 +16,7 @@ before_script:
   - bundle exec rake --version
   - bundle exec rake db:create db:migrate
 script:
-  - git diff --exit-code spec/dummy/db/structure.sql
+  # Disabling this check because it is proving unreliable
+  #- git diff --exit-code spec/dummy/db/structure.sql
```

Comment:
Should we look into why this is unreliable rather than disabling it?

Reply:
I approved this change in a discussion somewhere. I'd say just delete it.

This test relies on specific versions of Postgresql to be stable, which makes it super annoying, and it does not provide much value. We disabled a similar test in framework a few years ago.